src/kernel-install/kernel-install.c

```diff
@@ -983,6 +995,7 @@ static int context_build_environment(Context *c) {
                                  "KERNEL_INSTALL_LAYOUT",           context_get_layout(c),
                                  "KERNEL_INSTALL_INITRD_GENERATOR", strempty(c->initrd_generator),
                                  "KERNEL_INSTALL_UKI_GENERATOR",    strempty(c->uki_generator),
+                                 "KERNEL_INSTALL_BOOT_ENTRY_TYPE",  boot_entry_type_json_to_string(c->entry_type),
```

Comment:
should maybe be suppressed if not set

Reply:
It is unset by default:
`$ ./build/kernel-install remove test -v`
`...`
`  KERNEL_INSTALL_UKI_GENERATOR=`
`  KERNEL_INSTALL_STAGING_AREA=/tmp/kernel-install.staging.BIcKKg`
`...`